### DISCUSSION TITLE
Docker e IP

### DISCUSSION THREAD

**Discussion by masolo3000** [indomus/forum#2403] [Networking]
ciao ho seguito la guida Installare e configurare Caddy (reverse proxy) con Docker su Raspberry Pi OS tutto funziona perfettamente ,solo una cosa mi chiedo al momento che riavvio il sistema gli IP dei container docker variano per esempio caddy  da 172.17.0.2 a 172.17.0.5 ,come è possibile mantenere gli IP statici assegnati sempre allo stesso container?

**Comment by snakuzzo** [comment]
Puoi assegnare IP statici, ma devi studiare [QUI](https://docs.docker.com/engine/reference/commandline/network_create/).
Detto ciò, te ne puoi fregare di assegnare sempre ip statici. Se il tuo problema dovesse essere quello dei trusted proxy, puoi semplicemente mettere tra i trusted tutta la subnet... es. 172.17.0.0/16
